src/entities/forgot-password.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { User } from './user.entity';
+import { IsString, MaxLength } from 'class-validator';
+
+@Entity()
+export class ForgotPassword {
```

Comment:
quite complicated? why do you use a separate table for it? in your implementation I don't really see sense it in

Reply:
because in future the user table will be more complicated and like that it follows solid pattern